### PR TITLE
#64 - Removed base path support

### LINKED ISSUE #64: Remove base path support feature
Base path support is not required in PHP Composer. In Artipie it should be established on upper level

### PR DESCRIPTION
**Pull request by olegmoz**
Closes #64 
Removed base path support

### CODE REVIEW

File: src/test/java/com/artipie/composer/http/PhpComposerTest.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ void shouldGetPackageContent() throws Exception {
             data
         );
         final Response response = this.php.response(
-            new RequestLine(RqMethod.GET, "/base/p/vendor/package.json").toString(),
+            new RequestLine(RqMethod.GET, "/p/vendor/package.json").toString(),
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/composer/http/PhpComposerTest.java:88]: @olegmoz I'd suggest adding test method to verify that `METHOD_NOT_ALLOWED` is returned when request method is not `get` or `put`

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz one suggestion, take a look please

**Comment by olegmoz** [comment]
@olenagerasimova thanks, added tests for multiple methods

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks

**Comment by 0crat** [comment]
Job `gh:artipie/composer-adapter#65` is not assigned, can't get performer

**Comment by olenagerasimova** [comment]
@olegmoz can we release this adapter? Yegor already touched `.rultor`

**Comment by olegmoz** [comment]
@olenagerasimova there are still problems with release, see https://github.com/artipie/composer-adapter/issues/64

**Comment by olenagerasimova** [comment]
@olegmoz oh, I see, thanks

**Pull request merged**